src/components/InstancesList/InstancesList.js

```diff
@@ -193,6 +195,11 @@ class InstancesView extends React.Component {
             this.generateInstancesIdReport();
           }}
         >
+          <Icon
```

Comment:
I think we can try to add a test to check if the correct (save) icon is in place. @VictorSoroka1  @Yuliia-Dovhal  what do you think about it?

Reply:
The existed tests are broken and I will spend too much time for fixing all that tests before even adding my ones. Also it seems like fixing existing tests is not only my responsibility.
Maybe @VictorSoroka1 can disagree?

Reply:
Agree.